roughpy_jax/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+project(RoughPyJaxModule LANGUAGES CXX)
+
+# Optionally enable CUDA build
+include(CheckLanguage)
+check_language(CUDA)
+
+if (ROUGHPY_JAX_CUDA)
+    if (NOT DEFINED CMAKE_CUDA_COMPILER)
+        message(FATAL_ERROR "CMAKE_CUDA_COMPILER must be available for ROUGHPY_JAX_CUDA")
+    endif()
+
+    enable_language(CUDA)
+    set(CMAKE_CUDA_STANDARD 17)
+    set(CMAKE_CUDA_STANDARD_REQUIRED ON)
+    set(CMAKE_CUDA_SEPARABLE_COMPILATION ON)
+    if (NOT DEFINED CMAKE_CUDA_ARCHITECTURES)
+        set(CMAKE_CUDA_ARCHITECTURES 80 86 90 120)
+    endif()
+
+    find_package(CUDAToolkit REQUIRED)
+endif()
+
+# Get XLA includes via jax lib
+find_package(Python 3.8 REQUIRED COMPONENTS Interpreter Development.Module)
```

Comment:
Python should already be located from the top-level CMakelists

Reply:
Commented in 22f94d1 to pick up in parent branch